src/content/app/species/SpeciesPageContent.tsx

```diff
@@ -91,16 +96,29 @@ const SpeciesPageContent = () => {
     return null; // TODO: consider some kind of a spinner?
   }
 
+  const SidebarContent = () => {
+    const isSpeciesSidebarModalOpened = useAppSelector(
+      getIsSpeciesSidebarModalOpened
+    );
+
+    return isSpeciesSidebarModalOpened ? (
+      <SpeciesSidebarModal />
+    ) : (
+      <SpeciesSidebarOverview />
```

Comment:
Why do you call it an overview?

Reply:
Looking at other apps in case we have tabs in the future, I named it like this. What do you think?

Reply:
Please change this to `SpeciesPageSidebar`